src/sonic-bgpcfgd/staticroutebfd/main.py

```diff
@@ -0,0 +1,691 @@
+import os
```

Comment:
Any reason for adding this static route bfd under sonic-bgpcfgd folder? I believe, this is the generic module and no dependency on bgpcfgd. if you keep this module generic and keep it outside of sonic-bgpcfgd (may be one of the host services), we can make use of this for sonic-frr-mgmt-framework/frrcfgd as well.

Reply:
the StaticRouteBfd design watches and updates redis db tables only, it could be in anywhere in the system where can access redis db tables. 
Yes. this module has no dependency on bgpcfgd. we need to put it into a container to start (restart in the case of crash)  the process. the reason we put the code under sonic-bgpcfgd just because the logic is related. 
(originally we were planning to design the new component as one of the "manager" in the runner/manager framework. but later we found that the current manager watches one table only, but the StaticRouteBfd needs to watch both config db and state db. so we moved it out, run as a separate process.)

Reply:
also in multi-asic where we have multiple bgp docker instance it aligns with that model